### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by dieum512**
In this milestone I:

- Styled the project and
- Checked and Corrected some linter errors

Thank you for the time and energy to review this project

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Hi @dieum512 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project with failing linter checks. Kindly fix them and resubmit your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/books/Books.js [hunk 1]
```diff
@@ -5,10 +5,30 @@ const Book = ({
   category, title, author, remove,
 }) => (
   <>
-    <p>{category}</p>
-    <h2>{title}</h2>
-    <p>{author}</p>
-    <button type="button" onClick={remove}>Delete</button>
+    <div className="books-container">
```
Comment (hiromataba) [inline src/components/books/Books.js:8]: - [ ] Please Fix your category page to match the original design

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @dieum512    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — COMMENTED** [review]
Hi @dieum512, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project that has failing linters, Kindly make sure your linters are all passing before submitting the PR.

## Stlylint AutoFix
- [ ] if you have a lot of CSS line errors, I kindly suggest you fix them with `npx stylelint --fix "**/*.{css,scss}"`, but please get used to fixing the errors line by line also because it'll help you learn best practices.

## ESlint AutoFix
- [ ] if you have a lot of ESlint line errors, I kindly suggest you fix them with `npx eslint --fix`, but please get used to fixing the errors line by line also because it'll help you learn best practices.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @dieum512     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**